projects/angular/storyteller/src/app/routes/feed/feed.component.scss

```diff
@@ -1,6 +1,7 @@
 .page-container {
   >.align-container {
     >.feed {
+      width: 930px;
```

Comment:
Fixed width of feed block, to avoid its disappearing, when it doesn't contain any story.